resotocore/resotocore/cli/command.py

```diff
@@ -5092,6 +5092,190 @@ async def app_run(
+                ArgInfo(None, True, help_text="<to_graph_name>"),
+                ArgInfo("--force", False),
+            ],
+            "import": [
+                ArgInfo(None, True, help_text="<from_graph_name>"),
+                ArgInfo(None, True, help_text="<to_graph_name>"),
+                ArgInfo("--force", False),
+            ],
+        }
+
+    def parse(self, arg: Optional[str] = None, ctx: CLIContext = EmptyContext, **kwargs: Any) -> CLIAction:
+        async def graph_list(pattern: Optional[str]) -> AsyncIterator[JsonElement]:
+            graphs = await self.dependencies.graph_manager.list(pattern)
+            for graph in graphs:
+                yield graph
+
+        async def graph_copy(
+            source: Optional[GraphName], destination: GraphName, force: bool
+        ) -> AsyncIterator[JsonElement]:
+            if not source:
+                source = ctx.graph_name
+            await self.dependencies.graph_manager.copy(source, destination, replace_existing=force)
+            yield f"Graph {source} copied to {destination}."
+
+        async def graph_snapshot(source: Optional[GraphName], label: str) -> AsyncIterator[JsonElement]:
+            if not source:
+                source = ctx.graph_name
+            snapshot_name = await self.dependencies.graph_manager.snapshot(source, label)
+            yield f"Graph {source} snapshoted to {snapshot_name}."
+
+        async def graph_delete(graph_name: GraphName) -> AsyncIterator[JsonElement]:
+            await self.dependencies.graph_manager.delete(graph_name)
+            yield f"Graph {graph_name} deleted."
+
+        async def write_result_to_file(export_lines: AsyncIterator[str], file_name: str) -> AsyncIterator[str]:
+            temp_dir: str = tempfile.mkdtemp()
+            path = os.path.join(temp_dir, file_name)
+            try:
+                async with aiofiles.open(path, "w") as f:
+                    async for line in export_lines:
+                        await f.write(line + "\n")
+                yield path
+            finally:
+                shutil.rmtree(temp_dir)
+
+        async def graph_export(graph_name: Optional[GraphName], file_name: str) -> AsyncIterator[JsonElement]:
+            if not graph_name:
+                graph_name = ctx.graph_name
+            lines = await self.dependencies.graph_manager.export_graph(graph_name)
+            return write_result_to_file(lines, file_name)
+
+        async def graph_import(
+            graph_name: Optional[GraphName], file_name: str, force: bool
+        ) -> AsyncIterator[JsonElement]:
+            if not graph_name:
+                graph_name = ctx.graph_name
+
+            path = ctx.uploaded_files.get("dump")
+            if not path:
+                raise ValueError(f"File {file_name} was not uploaded.")
+
+            async with aiofiles.open(path, "r") as f:
+
+                async def iterator() -> AsyncIterator[str]:
```

Comment:
That's a python internal name - consider choosing a different one. e.g. `all_lines` or similar